### ISSUE TITLE
Rails 4.2.1 support

### ISSUE THREAD

**Issue by gwincr11** [Compass/compass-rails#236]
Hello,

I am trying to setup rails 4.2.1 support. I am running into trouble with sprockets compatibility on the master branch. Bundle install returns the following error:
```
  In snapshot (Gemfile.lock):
    sprockets (= 3.0.1)

  In Gemfile:
    sprockets (< 4.0, >= 2.8) ruby

    compass-rails (>= 0) ruby depends on
      sprockets (< 2.13) ruby
```
Is there a reason why compass rails does not support a higher version of sprockets? Any way I can get this installed and working?

Thanks,
Cory

**Comment by kushkella** [comment]
+1 I'm running into similar issues. compass-rails should support sprockets 3.0

**Comment by maurosbu** [comment]
Could you please release a new version that spports last version of sass-rails (5.0.3) and last version of sprockets (3.0.2)? Thanks.

**Comment by gwincr11** [comment]
I got this working
gem 'sass-rails'
gem "compass-rails", git: 'https://github.com/Compass/compass-rails', branch: 'master'

bundle update.

**Comment by zetsubo** [comment]
+1 been fighting this for a bit too now

**Comment by JacobEvelyn** [comment]
Me too.

**Comment by rmckayfleming** [comment]
Eagerly awaiting Sprockets 3 support.

**Comment by maurosbu** [comment]
+10

**Comment by zetsubo** [comment]
going to take a stab at a PR when I get a spare moment, I want an updated sprockets for babeljs so bad :<

**Comment by maurosbu** [comment]
I want an update of sprockets for chosen-rails.
There are a lot of gems with compass-rails as a runtime dependency, I hope a new release of compass-rails to be released soon.

**Comment by craigmcnamara** [comment]
If someone want's to take a stab at support for sprockets 3 support I'd merge it as compass-rails 3.0. I'm not currently supporting any projects with that requirement so it will be a while until I'm concerned enough to do the work.

**Comment by zetsubo** [comment]
sounds reasonable :3

**Comment by BartlomiejSkwira** [comment]
Same issue here. Version `2.0.1` doesn't have sprockets as a dependency at all, so bundle install will work, but later I get:
```
File to import not found or unreadable: compass.
```
Similar issues: https://github.com/Compass/compass-rails/issues/232 https://github.com/Compass/compass-rails/issues/229

**Comment by BartlomiejSkwira** [comment]
```
#Gemfile
gem 'sass-rails', '5.0.1'
gem 'compass', '1.0.3'
gem 'compass-rails', '2.0.4'
```
```sh
bundle update sass-rails compass-rails
```
..and it works :)

**Comment by vorant94** [comment]
Does anybody have another solution? Because It crashes with using sprockets-rails gem
````
Bundler could not find compatible versions for gem "sprockets":
  In Gemfile:
    compass-rails (>= 0) ruby depends on
      sprockets (< 2.13) ruby

    sprockets-rails (>= 0) ruby depends on
      sprockets (3.0.0)
````

**Comment by astery** [comment]
@BartlomiejSkwira, that's not resolve the issue, just degrades sprockets to < 2.13.

**Comment by astery** [comment]
I've looked for forks, @urkle make it work - https://github.com/urkle/compass-rails/commit/2fe981f33e6e3834484cfb58b0d01864735d4602. Why author not suggested as PR? Does it have flaws?

**Comment by BartlomiejSkwira** [comment]
@astery  Well yeah, it was just a workaround

**Comment by astery** [comment]
@BartlomiejSkwira, thanks for response. But it doesn't seem clear for me - you think degrading sprockets is workaround or @urkle patch is doesn't look as relaible and should not be merged?

**Comment by francois2metz** [comment]
Fixed now. Should be closed since #246 has been merged and the 3.0 released.
